source/content/php-versions.md

```diff
@@ -50,7 +50,19 @@ Click on the links above to see the complete PHP info for each version, includin
 
 <dd>
 
-End-of-life (**EOL**) versions are available on the platform but no longer under active development, and should not be used unless absolutely necessary.
+End-of-life (**EOL**) versions are available on the platform but no longer receiving updates, and should not be used unless absolutely necessary.
```

Comment:
Could we put the EOL and EOS icons next to their respective section headings?

Reply:
This seems like a good idea.